### PR TITLE
project card bug issue resolved 

### PR DESCRIPTION
**Pull request by jatinjain001**
I resolved the issue you can review my PR

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -24,7 +24,41 @@ let header = $(`
 </div>
 </nav>`);
 
-let footer = $(`
```
Comment (smaranjitghose) [inline assets/js/app.js:27]: Please do not change the files not related to the task your assigned to complete

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
navbar

**Comment by jatinjain001** [comment]
should i need to do again PR

**Comment by smaranjitghose** [comment]
> should i need to do again PR

If you are continuing with the contribution to the issue, then yes

**Comment by jatinjain001** [comment]
okay

**Comment by jatinjain001** [comment]
Okay I am reaally sorry

**Pull request closed without merging**